assetserver.go

```diff
@@ -360,11 +369,17 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	h := w.Header()
-	if tag == "" {
-		h.Set("Cache-Control", cacheControlUnversioned)
+	var cc string
+	if s.noCache {
+		cc = "no-cache"
 	} else {
-		h.Set("Cache-Control", cacheControlVersioned)
+		if tag == "" {
+			cc = "public, max-age=60"
+		} else {
+			cc = "public, max-age=31536000, immutable"
```

Comment:
I debated whether to set no-cache for tagged assets. I ended up going with it because (a) it's easier to explain the behavior, (b) I can't think of why you'd particularly want or need cache-forever in a dev context, which is what `NoNewCache` is made for, and (c) in theory these objects are just wasteful in the browser cache to the extent that they are cached for any long period of time (though I haven't investigated how browsers actually cache localhost stuff; it probably doesn't matter much in any case).

I don't feel strongly about it though.